commons-dlp/src/main/java/org/exoplatform/commons/dlp/rest/DlpItemRestServices.java

```diff
@@ -52,6 +55,29 @@ public Response getDlpPositiveItems(@ApiParam(value = "Offset", required = false
         }
     }
 
+    @GET
+    @Path("/itemUrl/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @RolesAllowed("administrators")
+    @ApiOperation(value = "Retrieves the url of dlp positive items", httpMethod = "GET", response = Response.class, produces = "application/json",
```

Comment:
Why do we need a specific rest service only for url ?
Is it not possible to get it when getting the item list ?